### PR TITLE
Improve converter install role

### PR DESCRIPTION
**Pull request by yakky**
Remove explicit references to pandoc, use converter instead

Add option to download binary commands and make it the default

### CODE REVIEW

**Comment by yakky** [comment]
Closing this, wrong target branch

File: roles/converter-filters/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  apt: pkg={{ item }} state=present update_cache=true install_recommends=no
+  with_items:
+    - libgmp3-dev
+  when: app == 1
+  tags:
+     - setup
+
+- name: setup checkout path
+  file: path="{{ converter_checkout_path }}" owner="{{ converter_user }}" state=directory
+  when: converter == 1 and converter_build == 1
```
Comment (ioggstream) [inline roles/converter-filters/tasks/main.yml]: consider blocks ;) @yakky https://www.google.it/search?q=ansible+blocks&oq=ansible+blocks&aqs=chrome..69i57j69i60j0l4.2392j0j7&sourceid=chrome&ie=UTF-8

File: roles/converter-filters/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+  tags:
+     - deploy_converter
+     - setup
+
+- name: clone converter
+  become_user: "{{ converter_user }}"
+  git: repo={{ converter_repo }} dest="{{ converter_checkout_path }}/docs-italia-comandi-conversione" version={{ converter_branch }} force={{ force }}
```
Comment (ioggstream) [inline roles/converter-filters/tasks/main.yml]: line too long

File: roles/converter-filters/tasks/main.yml [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+  when: converter == 1 and converter_build == 1
+  tags:
+     - deploy_converter
+     - setup
+
+- name: clone filters
+  become_user: "{{ converter_user }}"
+  git: repo={{ converter_filters_repo }} dest="{{ converter_checkout_path }}/docs-italia-converter-filters" version={{ converter_filters_branch }} force={{ force }}
```
Comment (ioggstream) [inline roles/converter-filters/tasks/main.yml]: missing quotes (") ?

File: roles/converter-filters/tasks/main.yml [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+  tags:
+     - deploy_converter
+     - setup
+
+- name: build conversion commands - expect a **long wait** before it's ready
+  become_user: "{{ converter_user }}"
+  command: chdir="{{ converter_checkout_path }}/docs-italia-comandi-conversione" stack build
+  when: converter == 1 and converter_build == 1
+  tags:
+     - deploy_converter
```
Comment (ioggstream) [inline roles/converter-filters/tasks/main.yml]: 3 spaces. Consider 2 or 0. 

@yakky did you try ansible-lint?

**Pull request closed without merging**